incubator/hnc/hack/prow-e2e/Dockerfile

```diff
@@ -7,20 +14,14 @@ ENV GO111MODULE=on
 # Build static binaries; otherwise go test complains.
 ENV CGO_ENABLED=0
 
-# Install all basic dependencies. Bash is there in case you need to debug.
-# Curl, Docker, git and make are not included in the golang image.
-RUN apk add --no-cache bash curl docker git make
-#  bash curl docker gcc git jq make openssh-client
-
 # Install Kind
 RUN go get sigs.k8s.io/kind@v0.9.0
 
-# Install kubectl and make sure it's available in the PATH.
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
-RUN chmod +x ./kubectl
-RUN mv ./kubectl /bin
-
```

Comment:
Why don't we need this any more? Or is it moved to somewhere else?

Reply:
`kubectl` is now included in the base image (`krte` vs `golang`).